Add clarification to the setup instructions (#66)
chanzuckerberg/sorbet-rails: README.md

1. Follow the steps [here](https://sorbet.org/docs/adopting) to set up the latest version of Sorbet, up to being able to run `srb tc`.

> Note: do not add `sorbet-rails` to your Gemfile yet, until you've set up sorbet. `sorbet-rails` add rbis that may [break](https://github.com/sorbet/sorbet/issues/1158) `srb init` until it is set up.

Reviewer: ```suggestion
Note: Do not add `sorbet-rails` to your Gemfile until after you've set up Sorbet. `sorbet-rails` adds RBI files that may [break](https://github.com/sorbet/sorbet/issues/1158) `srb init`.
```
